web/src/containers/activity/CostAllocateFFPQuarterly.js

```diff
@@ -151,48 +125,54 @@ class CostAllocateFFPQuarterly extends Component {
                       headers={`act_qbudget_fy${year} act_qbudget_fy${year}_subtotal`}
                     >
                       <Dollars>
-                        {quarterlyFFP[year].subtotal.combined.dollars}
+                        {quarterlyFFP[year].subtotal[name].dollars}
                       </Dollars>
                     </td>
                   </Fragment>
                 </tr>
-              </tbody>
-            </table>
-          </div>
-        ))}
-        <h3>{`Total FFY ${years[0]} - ${years[years.length - 1]}`}</h3>
-        <table className="table-cms table-fixed table-fit-contents">
-          <thead>
-            <tr>
-              <th />
-              <th>Total</th>
-            </tr>
-          </thead>
-          <tbody>
-            {['state', 'contractors'].map(name => (
-              <Fragment key={name}>
-                <tr
-                  key={name}
-                  className={`align-middle ${
-                    name === 'combined' ? 'bold' : ''
-                  }`}
-                >
-                  <th>{EXPENSE_NAME_DISPLAY[name]}</th>
-                  <td className="bold mono right-align">
-                    <Dollars>{quarterlyFFP.total[name]}</Dollars>
-                  </td>
-                </tr>
               </Fragment>
             ))}
-            <tr>
-              <th>{EXPENSE_NAME_DISPLAY.combined}</th>
-              <td className="bold mono right-align">
-                <Dollars>{quarterlyFFP.total.combined}</Dollars>
-              </td>
+            <tr className="bold">
+              <td>{EXPENSE_NAME_DISPLAY.combined}</td>
+              <Fragment key={year}>
+                {QUARTERS.map(q => (
+                  <td
+                    className="mono right-align"
+                    key={q}
+                    headers={`act_qbudget_fy${year} act_qbudget_fy${year}_q${q}`}
+                  >
+                    <Dollars>
+                      {quarterlyFFP[year][q].combined.dollars}
+                    </Dollars>
+                  </td>
+                ))}
+                <td
+                  className="bold mono right-align"
+                  headers={`act_qbudget_fy${year} act_qbudget_fy${year}_subtotal`}
+                >
+                  <Dollars>
+                    {quarterlyFFP[year].subtotal.combined.dollars}
+                  </Dollars>
+                </td>
+              </Fragment>
             </tr>
           </tbody>
         </table>
-      </div>
+        {year === years[years.length - 1] &&
```

Comment:
This felt a little hacky to me, but since we only want to render the total at the end of the quarterly section, and the quarterly section now gets rendered multiple times in multiple places, I wasn't sure how to do this without hooking up these props to flow in via `CostAllocate.js`.